src/python/pants/backend/debian/rules_integration_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from textwrap import dedent
+
+import pytest
+
+from pants.backend.debian.rules import package_debian_package
+from pants.backend.debian.target_types import DebianPackage
+from pants.backend.python import target_types_rules
+from pants.core.util_rules import source_files
+from pants.core.util_rules.source_files import SourceFilesRequest, SourceFiles
+from pants.engine.rules import QueryRule
+from pants.testutil.rule_runner import RuleRunner
+
+
+@pytest.fixture
+def rule_runner() -> RuleRunner:
+    return RuleRunner(
+        rules=[
+            *package_debian_package(),
```

Comment:
You probably instead want to do:

```
from pants.backend.debian.rules import rules as debian_rules
```

Then register `*debian_rules()`. Right now you're only registering that single rule, and you want to register every rule in the file.

Reply:
Done!